tests/swagger/test_validation.py

```diff
@@ -113,7 +113,7 @@ def test_validate_headers(self):
         for head in headers:
             self.request.headers = head
             self.assertRaises(ValidationError, unmarshal_request,
-                              self.request, self.resources['Buckets'].get_buckets)
+                              self.request, self.resources['Bucket'].get_buckets)
```

Comment:
That would be good. I mean, now we are generating it automatically I don't think we should test it this way. We can probably think of something better.